spark/src/test/scala/org/apache/spark/sql/delta/DeltaParquetFileFormatSuite.scala

```diff
@@ -33,26 +29,31 @@ import org.apache.parquet.hadoop.ParquetFileReader
 import org.apache.spark.sql.{DataFrame, Dataset, QueryTest}
 import org.apache.spark.sql.execution.datasources.{HadoopFsRelation, LogicalRelation}
 import org.apache.spark.sql.test.SharedSparkSession
-import org.apache.spark.util.{SerializableConfiguration, Utils}
+import org.apache.spark.util.SerializableConfiguration
 
 class DeltaParquetFileFormatSuite extends QueryTest
-  with SharedSparkSession with DeltaSQLCommandTest {
+  with SharedSparkSession
+  with DeletionVectorsTestUtils
+  with DeltaSQLCommandTest {
   import testImplicits._
 
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    enableDeletionVectorsInNewTables(spark.conf)
```

Comment:
Why enable DV for all tests?

Reply:
@xupefei What do you mean sorry? Could you further elaborate?

Reply:
@xupefei I added some tests with DV turned off, would you be so kind to take another look at the PR :)?